css/style.css

```
@@ -481,70 +482,73 @@ img {
 }
 
 .google_map {
-  position: relative;
-  z-index: 1;
-  overflow: hidden;
-  width: 1199px;
+  position: absolute;
+  width: 1200px;
+  height: 594px;
+}
+
+.google-img {
+  z-index: 2;
 }
 
 /* Footer */
 
 .main-footer {
-  background-color: var(--basic-white);
-  opacity: 0.9;
+  background-color: var(--background-footer);
   color: var(--basic-black);
   display: grid;
-  grid-template-columns: 73px 327px 1fr 327px 73px;
+  grid-template-columns: 400px 400px 400px;
+  grid-template-rows: min-content;
+  align-items: center;
+  padding: 20px 0px;
+  margin-top: 35px;
   height: 120px;
```

Лучше не задавать фиксированную высоту, а использовать паддинги или **min-height**, это нужно для того, чтобы [при переполнении контентом](https://up.htmlacademy.ru/htmlcss/30/criteries#d19) блоки не сломались
```suggestion
  grid-template-columns: 1fr 1fr 1fr;
  align-items: center;
  padding: 20px 0px;
```